opal/mca/base/help-mca-base.txt

```diff
@@ -59,3 +59,9 @@ all components *except* a and b", while "c,d" specifies the inclusive
 behavior and means "use *only* components c and d."
 
 You cannot mix inclusive and exclusive behavior.
+[framework-param:exclude-always-component]
+A request was made to exclude a component from consideration that
+must always be considered.
+
```

Comment:
My $0.02: if someone tries to disable it and it can't be disabled, we should abort (in line with the philosophy of "a human asked for X, and OMPI can't provide X, so OMPI needs to abort and let a human figure it out").